Makefile

```diff
@@ -74,7 +74,7 @@ requires the TOIL_DOCKER_REGISTRY variable to be set to a value other than the d
 accidentally pushing to the official Docker registry for Toil.
 
 The TOIL_DOCKER_NAME environment variable can be set to customize the appliance image name that
-is created by the 'docker' target and pushed by the 'push_docker' target. The Toil team's
```

Comment:
why escape this one and not the rest? to satisfy syntax highlighting?

Reply:
Because it is not matched, so emacs thinks the rest of the makefile is a quote.  Didn't really care about syntax highlighting in the help message.   I have never seen define used this way, so it is not surprising it doesn't understand it.